src/main/java/seedu/quotesify/commands/DeleteCommand.java

```diff
@@ -294,6 +304,17 @@ private void clearBookmarkFromDeletedBook(Book targetBook, BookmarkList bookmark
         }
     }
 
+    private int convertBookmarkIndexToInt(String indexString, TextUi ui) {
+        int index = -1;
+        try {
+            index = Integer.parseInt(information);
```

Comment:
Just to confirm, will your bookmark work if you do not 0-index this?

Reply:
Oh I see how your code works now. As a suggestion, it will be better to 0-index the number here, so that you will not need to -1 from other places of your code such as your delete(). 

I will assume that your code is working and approve it.

Reply:
thanks for the suggestion!